challenge.md

```diff
@@ -7,37 +7,38 @@ Write your answers inside this file, where it's indicated by the comments.
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
 <!-- Answer Starts Here -->
+git clone [clone URL]
 
 <!-- Answer Ends Here -->
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
 <!-- Answer Starts Here -->
-
+git status
```

Comment:
Well, yes - this will show you what files have been created, removed, moved, or otherwise changed. But it won't show you how the content of those files has changed. To do that, you'd need to use `git diff` (or, as you pointed out below, `git show`).